# Set the LD_LIBRARY_PATH to an empty string because it affects
# the way the test harness scripts are being run.
# bash --noprofile is documented.
# Not for upstream. They don't run their test harness the way we do.
--- ltmain.sh	2014-10-26 00:35:44.000000000 -0700
+++ ltmain.sh	2015-07-16 09:51:16.867440561 -0700
@@ -5094,7 +5094,9 @@
 	func_emit_wrapper_arg1=${1-no}
 
 	$ECHO "\
-#! $SHELL
+#!/bin/bash --noprofile
+
+set -o posix
 
 # $output - temporary wrapper script for $objdir/$outputname
 # Generated by $PROGRAM (GNU $PACKAGE) $VERSION
@@ -5123,6 +5125,8 @@
 BIN_SH=xpg4; export BIN_SH # for Tru64
 DUALCASE=1; export DUALCASE # for MKS sh
 
+export LD_LIBRARY_PATH=""
+
 # The HP-UX ksh and POSIX shell print the target directory to stdout
 # if CDPATH is set.
 (unset CDPATH) >/dev/null 2>&1 && unset CDPATH
